.travis.yml

```diff
@@ -15,27 +15,14 @@ before_install:
 bundler_args: "--binstubs --standalone --without documentation --path ../bundle"
 script: "script/run_build"
 rvm:
-  - 1.8.7
```

Comment:
Really you can just delete this file, we won't be using travis for RSpec 4's CI.

Reply:
That's for sure, once it's all done.
But before it was done, I preferred to have at least some builds to constantly updating two repos I was actively committing to to `drop-old-rubies` to those frequent commits to `4-0-dev`.
Once it's all settled, I'll go through all the changes thoroughly again myself and will ask for a detailed review.